app/routes/products.$categorySlug/route.module.scss

```diff
@@ -28,6 +28,11 @@
     margin-bottom: 12px;
 }
 
+.categoryList {
+    list-style: none;
+    padding: 0;
+}
```

Comment:
What about margin? `UL` elements also have margin by default.

Reply:
The new reset takes care of margin